### PR TITLE
Do not throw if initialized by Winforms UI thread

### PR DESCRIPTION
**Pull request by jmarolf**
Simplest fix for https://github.com/dotnet/roslyn/issues/10510 to be considered for Micro-Update.

Verified that this fixes the issue on the repro machine.  PowerBI team verified that this fixes the issue

### CODE REVIEW

**Comment by jmarolf** [comment]
@jasonmalinowski @CyrusNajmabadi @Pilchie 
@dotnet/roslyn-ide

**Comment by jmarolf** [comment]
test vsi please

File: src/Workspaces/Core/Portable/Utilities/ForegroundThreadDataKind.cs [hunk 1]
```diff
@@ -43,6 +44,10 @@ internal static ForegroundThreadDataKind CreateDefault(ForegroundThreadDataKind
 
                     return JoinableTask;
 
+                case "System.Windows.Forms.WindowsFormsSynchronizationContext":
+
```
Comment (CyrusNajmabadi) [inline src/Workspaces/Core/Portable/Utilities/ForegroundThreadDataKind.cs]: Unnecessary newline.

**Comment by CyrusNajmabadi** [comment]
:+1:

File: src/Workspaces/Core/Portable/Utilities/ForegroundThreadDataKind.cs (hunk 1 above)
Comment (jmarolf) [inline src/Workspaces/Core/Portable/Utilities/ForegroundThreadDataKind.cs]: While unnecessary, it is consistent with the styling for the rest of the file.

Comment (CyrusNajmabadi) [inline src/Workspaces/Core/Portable/Utilities/ForegroundThreadDataKind.cs]: Fix the rest of the file :)

**Pull request closed without merging**